docs/modules/operation/pages/geographical-map/searching.adoc

```diff
@@ -0,0 +1,73 @@
+# Map Search
+
+The search bar lets you do a simple text search, as well as searching particular fields associated with the node.
+
+### Search Fields
+
+You can search the following fields:
+
+[options="header"]
+[cols="2*"]
+|====
+| Field
+| Description
+
+| nodeId
+| The ID of the node.
```

Comment:
```suggestion
| The ID of the node
```
Please remove period from all of the description text. Apologies for missing this the first time.

Reply:
Done within https://github.com/OpenNMS/opennms/pull/4351/commits/be50829f2b9ad522e5e996322ae0489bb873c5cb